tests/bdd/features/health_probes/readiness_probe.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Readiness Probe
+
+  Background:
+    Given a running agent-core service
+    And a running REST service with "--core-health-freq" set to "10s"
```

Comment:
Please use milliseconds here, no need to wait so long for this test

Reply:
Okay.